### ISSUE TITLE
npm start & fails with strict mode

### ISSUE THREAD

**Issue by groovecoder** [rhelmer/tracking#2]
I tried following the README top-to-bottom exactly, and I got this ...

```
/Users/lcrouch/code/stop-tracking/server.js:3
let PORT = process.env.PORT || 5000;
^^^
SyntaxError: Unexpected strict mode reserved word
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

**Comment by rhelmer** [comment]
Huh! It WFM, what version of node? I have v5.3.0

**Comment by rhelmer** [comment]
Just upgraded to v5.9.1, do you see `"use strict";` at the top of `server.js`?

**Comment by groovecoder** [comment]
Ah, yeah I was on an older version of node. Maybe "requires node 5.x" on the README?

**Comment by rhelmer** [comment]
Fixed in PR #3

**Comment by rhelmer** [comment]
Thanks!
